localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -546,6 +587,26 @@ void NDTScanMatcher::publish_initial_to_result_distances(
     make_float32_stamped(sensor_ros_time, initial_to_result_distance_new));
 }
 
+void NDTScanMatcher::remove_ground_point_cloud(
+  const pcl::shared_ptr<pcl::PointCloud<PointSource>> & sensor_points_mapTF_ptr,
+  const pcl::shared_ptr<pcl::PointCloud<PointSource>> & no_ground_points_mapTF_ptr)
```

Comment:
```suggestion
  const pcl::shared_ptr<pcl::PointCloud<PointSource>> & no_ground_points_mapTF_ptr) const
```
I suggest adding const for this function

(or maybe you can make this as a separate function in `utils.cpp`, since it does not seem necessary to include it in NDTScanMatcher class.)

Reply:
Thank you very much.
If I remove the ground directly using Z-axis information, I will remove this function.
Now I am considering how to modify the algorithm for removing the ground.